frontend/src/pages/UploadPhoto.jsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from 'react';
+import AddPhotoAlternateIcon from '@mui/icons-material/AddPhotoAlternate';
+import ArrowBackIosIcon from '@mui/icons-material/ArrowBackIos';
+import ArrowForwardIosIcon from '@mui/icons-material/ArrowForwardIos';
+import ArrowBackIcon from '@mui/icons-material/ArrowBack';
+import EditIcon from '@mui/icons-material/Edit';
+import { useNavigate } from "react-router-dom";
+
+function UploadPhotos() {
+  const [imageFiles, setImageFiles] = useState([]);
+  const [isEditMode, setIsEditMode] = useState(false); 
+  const navigate = useNavigate(); 
+
+
+  const handleFileUpload = (event) => {
+      const file = event.target.files[0];
+      if (file && file.type.startsWith("image/")) { 
+          const reader = new FileReader();
+          reader.onload = function (e) {
+              setImageFiles([...imageFiles, { name: file.name, url: e.target.result }]);
+          };
+          reader.readAsDataURL(file); 
+      } else {
+          alert('Please upload a valid image file.');
+      }
+  };
+
+  const handleRemoveImage = (index) => {
+    const updatedImages = imageFiles.filter((_, i) => i !== index); 
+    setImageFiles(updatedImages);
+  };
+
+  const toggleEditMode = () => {
+    setIsEditMode(!isEditMode);
+  };
+
+  const handleBack = () => {
+    navigate("/newLog");
+  };
+
+  return (
+    <div className="upload-photos-container">
+      <div className="upload-box">
+        <button className="back-button" onClick={handleBack}>
+          <ArrowBackIcon />
+        </button>
+        <h1>Upload Photos</h1>
+        
+        <label htmlFor="fileInput" className="upload-area">
+          <input 
+            type="file" 
+            id="fileInput" 
+            accept="image/*"
```

Comment:
```suggestion
            accept="image/*,.pdf"
```

Do we want to accept PDFs as well, or only images? Maybe that's more of a question for the designers 😅